src/main/app/src/app/core/websocket/websocket.service.ts

```diff
@@ -138,18 +140,19 @@ export class WebsocketService implements OnDestroy {
     }
 
     public addListener(opcode: Opcode, objectType: ObjectType, objectId: string, callback: EventCallback): WebsocketListener {
-        const event: ScreenEvent = new ScreenEvent(opcode, objectType, objectId);
+        const event: ScreenEvent = new ScreenEvent(opcode, objectType, new ScreenEventId(objectId));
         const listener: WebsocketListener = this.addCallback(event, callback);
         this.send(new SubscriptionMessage(SubscriptionType.CREATE, event));
-        console.debug('listener added: ' + listener.key);
+        console.log('listener added: ' + listener.key);
```

Comment:
Zelf zou ik kiezen voor debug, je niet altijd geïnteresseerd in deze log regels, als je ze wil zien kan je in chrome log level verbose toevoegen